google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/FlexibleFacetEditor.java

```diff
@@ -270,6 +272,13 @@ public void onFacetInitialized(@NotNull Facet facet) {
       ((AppEngineFlexibleFacet) facet).getConfiguration().setAppYamlPath(appYamlField.getText());
       ((AppEngineFlexibleFacet) facet).getConfiguration().setDockerDirectory(
           dockerDirectoryField.getText());
+
+      // Called on explicitly adding the facet through Project Settings -> Facets, but not on the
+      // Framework discovered "Configure" popup.
+      UsageTrackerProvider.getInstance()
```

Comment:
this is not the right place for this; this called when opening (or maybe its closing..) the facet config. Use AppEngineStandard classes for a guide on where to add these